### ISSUE TITLE
fieldset aria-describedby not read when first field is a checkbox

### ISSUE THREAD

**Issue by ChristAsselin** [nvaccess/nvda#11617]
### Steps to reproduce:
Focus on the first checkbox does read the legend, but the element targeted by aria-describedby is not read.
```html
<fieldset aria-describedby="desc1">
   <legend>Vegetables</legend>
   <div id="desc1">Choose at most 2 vegetables.</div>
   <label>
      Tomato
      <input type="checkbox">
   </label>
   <label>
      Potato
      <input type="checkbox">
   </label>    
   <label>
      Avocado
      <input type="checkbox">
   </label>     
</fieldset>
```
https://output.jsbin.com/cawumusoqo/1 (examples for both checkbox and radio)

### Actual behavior:
The element targeted by attribute aria-describedby is not read when focus is on the first checkbox of the fieldset (or the last)

### Expected behavior:
The element targeted by aria-describedby is read for the first input type checkbox when the element gets focus. It should be the same behavior as the radio elements part of a fieldset.

#### NVDA version: 
2020.2

#### Windows version: 
10.0.19041 Build 19041

#### Name and version of other software in use when reproducing the issue:
Chrome 85.0.4183.102
Firefox 80.0.1

#### Other information about your system:
64 bits

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
